vale/styles/jargonLint/Technology.yml

```diff
@@ -0,0 +1,313 @@
+  Gulp: Gulp
+  H5: H5
+  Hadoop: Hadoop
+  HAProxy: HAProxy
+  HBase: HBase
+  HDFS: HDFS
+  Highcharts: Highcharts
+  HKEX: HKEX
+  Homebrew: Homebrew
+  (?<!\.)HTML: HTML  # also a file extension
+  (?<!\.)HTMX: HTMX  # also a file extension
+  HTTP(?!:): HTTP
+  HTTPS(?!:): HTTPS
+  Hugo: Hugo
+  I10n: I10n
+  i18n: i18n
+  IDE: IDE
+  iMovie: iMovie
+  IndexedDB: IndexedDB
+  Instagram(?!\.com): Instagram
+  Intel: Intel
+  iOS: iOS
+  iPad: iPad
+  iPadOS: iPadOS
+  iPhone: iPhone
+  IPsec: IPsec
+  IP[ -]?v6: IPv6
+  (?<!\.)ISO: ISO  # also a file extension
+  iTunes: iTunes
+  java[ -]?scripts?: JavaScript
+  Jekyll: Jekyll
+  Jenkins: Jenkins
+  Jest: Jest
+  JetBrains: JetBrains
+  Jira: Jira
+  jQuery: jQuery
+  (?<!\.)JSON[ -]?5: JSON5  # also a file extension
+  (?<!\.)JSON[ -]C: JSONC  # also a file extension
+  (?<!\./)json: JSON  # also a file extension
+  (?<!\.)JSON[ -]?LD: JSON-LD  # also a file extension
+  (?<!\.)JSONL: JSONL  # also a file extension
+  (?<!\.)JSON[ -]?RPC: JSON-RPC  # also a file extension
+  JWT: JWT
+  Kafka: Kafka
+  Kibana: Kibana
+  KO: KO
+  LinkedIn(?!\.com): LinkedIn
+  Linux: Linux
+  LocalStorage: LocalStorage
+  Lodash: Lodash
+  MacBook: MacBook
+  mac ?OS: macOS
+  MailChimp: MailChimp
+  Makefile: Makefile
+  man[ -]?page: man page
+  Markdown: Markdown
+  Microsoft: Microsoft
+  Modernizr: Modernizr
+  Moment\.?js: Moment.js
+  MongoDB: MongoDB
+  Mozilla: Mozilla
+  MVC: MVC
+  MVP.css: MVP.css
+  MySQL(?!\.com): MySQL
+  Nasdaq: Nasdaq
+  (?<!\.)ND[ -]?JSON: NDJSON  # also a file extension
+  NestJS: NestJS
+  Netflix: Netflix
+  Netlify: Netlify
+  Next.js: Next.js
+  Node[ .]?js: Node.js
+  Nominatim: Nominatim
+  NoSQL: NoSQL
+  npm: npm
+  Nvidia: Nvidia
+  NYSE: NYSE
+  OAuth: OAuth
+  Objective[\- ]C: Objective-C
+  Observability: Observability
+  OK: OK
+  OLAP: OLAP
+  OpenAPI: OpenAPI
+  OpenID[ -]?Connect: OpenID Connect
+  OpenSSL: OpenSSL
+  OSM: OSM
+  OSS: OSS
+  OVH(?!\.(?:com|net)): OVH
+  P2P: P2P
+  PaaS: PaaS
+  Parcel: Parcel
+  PayPal(?!\.com): PayPal
+  (?<!\.)PDF: PDF  # also a file extension
+  Perconn?a: Percona
+  Perl: Perl
+  (?<!\.)PHP: PHP  # also a file extension
+  Pinterest(?!\.com): Pinterest
+  Playstation: Playstation
+  (?<!\.)PNG: PNG  # also a file extension
+  pnpm: pnpm
+  Postgres? ?SQL: PostgreSQL
+  Postman: Postman
+  PostScript: PostScript
+  Prettier: Prettier
+  Prometheus: Prometheus
+  Pwned: Pwned
+  RabbitMQ: RabbitMQ
+  React Bootstrap: React Bootstrap
+  React Hook Form: React Hook Form
+  React[ \.]js: React
+  React Native: React Native
+  React: React
+  React-Toastify: React-Toastify
+  READ[ -]?ME(?:s): README
+  Reddit(?!\.com): Reddit
+  Redis: Redis
+  Redux: Redux
+  Redux-Saga: Redux-Saga
+  Redux Toolkit: Redux Toolkit
+  RefluxJS: RefluxJS
+  Renren: Renren
+  RESTful: RESTful
+  REST: REST
+  RFC: RFC
+  RSS: RSS
+  RubyGem: RubyGem
+  RubyGems: RubyGems
+  Rust: Rust
+  SaaS: SaaS
+  Sales?[ -]forces?: Salesforce
```

Comment:
@ccoVeille does the rule regarding ``Salesforce`` correct ``salesforce`` on your system. It doesn't seem to in mine and I'm not sure why.

Reply:
Right, just got confused with something else. can you add a question to the middle part, as in ``[ -]?`` so that it also catches ``salesforce`` with the missing capitalization.

Reply:
it works because there is `ignorecase:true` I kept the capital S because that's the way to refer to Salesforce

Reply:
I understood that part. I suggested changes so that ``salesforce`` would also be corrected.  Please go over them when you can.

Reply:
I will double-check, because now I get you said the current regexp doesn't while it should

Reply:
oh you spotted the error here https://github.com/jargonLint/jargonLint/pull/4#discussion_r1657246078